examples/state_space_connectivity.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun 14 13:15:43 2022
+
+@author: jordandrew
+
+For 'mne-connectivity/examples/' to show usage of MEGLDS 
+Use MNE-sample-data for auditory/left
```

Comment:
See how we construct examples/tutorials with proper titles etc.

Reply:
The docstring sections on tutorials seem to just be authors and a statement about the license. Happy to edit the author statement, but unsure if I should include "License BSD (3-Clause))" like in other tutorials. Thoughts?

Reply:
license is implied / specified at the repo level, no need to include it in a tutorial doc.